easybuild/easyconfigs/i/icc/icc-2016.0.109-GCC-4.9.3.eb

```diff
@@ -0,0 +1,29 @@
+# This is an easyconfig file for EasyBuild, see http://hpcugent.github.io/easybuild
+
+name = 'icc'
+version = '2016.0.109'
+
+homepage = 'http://software.intel.com/en-us/intel-compilers/'
+description = "C and C++ compiler from Intel"
+
+toolchain = {'name': 'dummy', 'version': 'dummy'}
+
+sources = ['parallel_studio_xe_%(version_major)s_composer_edition_for_cpp.tgz']
+
+checksums = ['f57a892fb494db3c80f20a88aa3e901f']
+
+gcc = 'GCC'
+gccver = '4.9.3'
+versionsuffix = '-%s-%s' % (gcc, gccver)
+
+dependencies = [(gcc, gccver)]
+
+components = ['intel-comp.*', 'intel-ccomp.*', 'intel-icc-.*', 'intel-openmp.*', 'intel-ipsc?.*']
```

Comment:
No, they are not needed. We only match, we don't use the result. The last regex should be `intel-ipsc?_`